ext/crypto/import_key.rs

```diff
@@ -75,6 +92,89 @@ pub fn op_crypto_import_key(
   }
 }
 
+macro_rules! decode_jwt_b64_int {
+  ($name:ident, $b64:expr, $err:expr) => {
+    let bytes = base64::decode_config($b64, base64::URL_SAFE)
+      .map_err(|_| data_error($err))?;
+    let $name = UIntBytes::new(&bytes).map_err(|_| data_error($err))?;
+  };
+}
```

Comment:
This can not be turned into a function, because lifetime issues that can not be resolved without hacks or GATs.

Reply:
I changed the name to `jwt_b64_int_or_err` (short enough to still keep all invocations 1 liners, but gets the point across).